docs/source/usage.rst

```diff
@@ -0,0 +1,164 @@
+.. _usage:
+
+Using ``repo2docker``
+=====================
+
+The core feature of repo2docker is to fetch a repo (from github or locally),
+build a container image based on the specifications found in the
+repo & optionally launch a local Jupyter Notebook you can use to explore it.
```

Comment:
The core functionality...GitHub or other locations) and build a container...in the repo.  Optionally, it can launch...Notebook which you...